components/layout/wrapper.rs

```diff
@@ -114,14 +114,14 @@ impl<T: ThreadSafeLayoutNode> ThreadSafeLayoutNodeHelpers for T {
             let style = self.as_element().unwrap().resolved_style();
 
             return match style.as_ref().get_counters().content {
-                content::T::Items(ref value) if !value.is_empty() => {
-                    TextContent::GeneratedContent((*value).clone())
+                Content::Items(ref value) if !value.is_empty() => {
+                    TextContent::GeneratedContent((*value).clone().to_vec())
```

Comment:
I think just `value.to_vec()` should work, shouldn't it?

Reply:
Yep, nice catch, forgot to remove that :).